src/connect4/core.rs

```diff
@@ -525,16 +494,25 @@ pub struct GameState {
     mouse_disabled: bool,
     gameover: bool,
     pub reset_button: Button,
-
+    pub main_menu_button: Button,
 }
 
 //Implementation based on structure in example from GGEZ repo (see https://github.com/ggez/ggez/blob/master/examples/02_hello_world.rs)
 impl GameState {
     pub fn new(ctx: &mut Context, players: i32) -> GameState {
         let board_pos = BOARD_POS_OFFSET;
-        let text = graphics::Text::new("Reset");
-        let text_width = text.width(ctx) as f32;
-        let text_height = text.height(ctx) as f32;
+        let main_menu_btn_text = graphics::Text::new(("Main Menu", graphics::Font::default(), 16f32,));
+        let main_menu_text_width = main_menu_btn_text.width(ctx) as f32;
+        let main_menu_text_height = main_menu_btn_text.height(ctx) as f32;
+        let main_menu_btn_outline = graphics::Rect::new(RESET_BUTTON_OFFSET.0 as f32, RESET_BUTTON_OFFSET.1 as f32 + main_menu_text_height, main_menu_text_width, main_menu_text_height);
+        let mut main_menu_btn = Button::new(main_menu_btn_text, main_menu_btn_outline);
+
+        let reset_text = graphics::Text::new(("Reset", graphics::Font::default(), 16f32,));
+        let reset_outline = graphics::Rect::new(RESET_BUTTON_OFFSET.0 as f32, RESET_BUTTON_OFFSET.1 as f32 + main_menu_text_height*3.0, main_menu_text_width, main_menu_text_height);
+        let mut reset_btn = Button::new(reset_text, reset_outline);
+
+        reset_btn.set_colors(MyColor::Brown, MyColor::Red);
+        main_menu_btn.set_colors(MyColor::Brown, MyColor::Green);
```

Comment:
Also while I like the color consistency, I might have the buttons be red/green backgrounds and something else for highlighting. Hard to distinguish them and font is small. But doesn't affect my willingness to merge this